source/docs/drivers.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Drivers"
+---
+
+Kitchen supports a driver plugin architecture which allows a user to run code
+on a variety of public cloud providers and local virtualization technologies.
+
+Some of these are included in `chef-dk` and some are not.
+
+There are drivers not included in this list, and it is not intended to be
+exhaustive. However, as a sample of some of the supported platforms:
+
+- kitchen-vagrant: Vagrant driver for Kitchen
```

Comment:
I only have it for some of them. Why don't I just have them all link to readmes here too? Updating momentarily.